instrumentation/spring/spring-webflux/spring-webflux-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/v5_0/client/WebClientHelper.java

```diff
@@ -5,38 +5,21 @@
 
 package io.opentelemetry.javaagent.instrumentation.spring.webflux.v5_0.client;
 
-import static java.util.Collections.singletonList;
-
 import io.opentelemetry.api.GlobalOpenTelemetry;
-import io.opentelemetry.instrumentation.api.incubator.semconv.http.HttpClientPeerServiceAttributesExtractor;
 import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
-import io.opentelemetry.instrumentation.spring.webflux.v5_3.internal.ClientInstrumenterFactory;
 import io.opentelemetry.instrumentation.spring.webflux.v5_3.internal.WebClientHttpAttributesGetter;
 import io.opentelemetry.instrumentation.spring.webflux.v5_3.internal.WebClientTracingFilter;
-import io.opentelemetry.javaagent.bootstrap.internal.AgentCommonConfig;
+import io.opentelemetry.javaagent.bootstrap.internal.JavaagentHttpClientInstrumenters;
 import java.util.List;
-import java.util.function.Function;
 import org.springframework.web.reactive.function.client.ClientRequest;
 import org.springframework.web.reactive.function.client.ClientResponse;
 import org.springframework.web.reactive.function.client.ExchangeFilterFunction;
 
 public final class WebClientHelper {
 
   private static final Instrumenter<ClientRequest, ClientResponse> instrumenter =
-      ClientInstrumenterFactory.create(
-          GlobalOpenTelemetry.get(),
-          builder ->
-              builder
-                  .setCapturedRequestHeaders(AgentCommonConfig.get().getClientRequestHeaders())
-                  .setCapturedResponseHeaders(AgentCommonConfig.get().getClientResponseHeaders())
-                  .setKnownMethods(AgentCommonConfig.get().getKnownHttpRequestMethods()),
-          builder -> builder.setKnownMethods(AgentCommonConfig.get().getKnownHttpRequestMethods()),
-          Function.identity(),
-          singletonList(
-              HttpClientPeerServiceAttributesExtractor.create(
-                  WebClientHttpAttributesGetter.INSTANCE,
-                  AgentCommonConfig.get().getPeerServiceResolver())),
-          AgentCommonConfig.get().shouldEmitExperimentalHttpClientTelemetry());
+      JavaagentHttpClientInstrumenters.create(
+          "io.opentelemetry.spring-webflux-5.3", WebClientHttpAttributesGetter.INSTANCE);
```

Comment:
should use `spring-webflux-5.0` here?

Reply:
the value comes from https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/spring/spring-webflux/spring-webflux-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webflux/v5_3/internal/ClientInstrumenterFactory.java#L37 - maybe it was wrong

Reply:
since this is not used to enable this instrumentation, we can probably change it

Reply:
server already uses 5.0

Reply:
this is what I did :smile: